certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1557 @@
+        "assignedLanguage": {
+          "description": "Gets or Sets AssignedLanguage",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.StandardLanguage"
+          }
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TitleOrderPolicy": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "attributes": {
+          "description": "Gets or Sets Attributes",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "ownerPolicies": {
+          "description": "Gets or Sets OwnerPolicies",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.TitleOrderOwnerPolicyDetail"
+          }
+        },
+        "loanPolicies": {
+          "description": "Gets or Sets LoanPolicies",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.TitleOrderLoanPolicyDetail"
+          }
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TitleOrderPartyRole": {
+      "description": "Gets or Sets TitleOrderPartyRole",
+      "type": "string"
+    },
+    "PubCS.TitleOrder.Api.Models.TitleOrderName": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "attributes": {
+          "description": "Gets or Sets Attributes",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "gender": {
+          "description": "Gets or Sets Gender",
+          "type": "string"
+        },
+        "dob": {
+          "description": "Gets or Sets Dob",
+          "type": "string"
+        },
+        "taxId": {
+          "description": "Gets or Sets TaxId",
+          "type": "string"
+        },
+        "foreignTaxId": {
+          "description": "Gets or Sets ForeignTaxId",
+          "type": "boolean"
+        },
+        "business": {
+          "description": "Gets or Sets Business",
+          "type": "string"
+        },
+        "title": {
+          "description": "Gets or Sets Title",
+          "type": "string"
+        },
+        "first": {
+          "description": "Gets or Sets First",
+          "type": "string"
+        },
+        "middle": {
+          "description": "Gets or Sets Middle",
+          "type": "string"
+        },
+        "last": {
+          "description": "Gets or Sets Last",
+          "type": "string"
+        },
+        "suffix": {
+          "description": "Gets or Sets Suffix",
+          "type": "string"
+        },
+        "full": {
+          "description": "Gets or Sets Full",
+          "type": "string"
+        },
+        "formal": {
+          "description": "Gets or Sets Formal",
+          "type": "string"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TitleOrderParty": {
+      "description": "",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.